src/NuGet.Core/NuGet.PackageManagement/Utility/Constants.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Please revert this.

Reply:
There is nothing to revert. This file is identical in both my branch, and in the parent.

Reply:
It's a line ending issue.

Do `git checkout dev --  src/NuGet.Core/NuGet.PackageManagement/Utility/Constants.cs`

Reply:
Fixed.